wire-ios-utilities/Source/Public/Data+ZMSCrypto.swift

```diff
@@ -78,8 +79,45 @@ public extension Data {
         return (self as NSData).zmDecryptPrefixedIV(withKey: key)
     }
 
-    func zmEncryptPrefixingPlainTextIV(key: Data) -> Data {
-        return (self as NSData).zmEncryptPrefixingPlainTextIV(withKey: key)
+    func zmEncryptPrefixingPlainTextIV(key: Data) -> Data? {
+
+        let keyLength = key.count
+        guard keyLength == kCCKeySizeAES256 else {
+            return nil
+        }
```

Comment:
Returning `nil` won't help the caller know what went wrong, should we consider making this a throwing function instead?